tests/datavolume_test.go

```diff
@@ -188,4 +236,50 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 			}
 		})
 	})
+
+	Describe("Verify DataVolume", func() {
+		table.DescribeTable("should", func(name, command, url, dataVolumeName, eventReason string, phase cdiv1.DataVolumePhase) {
```

Comment:
How is this different from the code above? it looks like a duplicate, its also failing due to the pvc being created against a storage class that cannot do block volume.
```
  Normal   WaitForFirstConsumer  3s (x9 over 1m)  persistentvolume-controller  waiting for first consumer to be created before binding
  Warning  VolumeMismatch        3s (x7 over 1m)  persistentvolume-controller  Cannot bind PersistentVolume "local-pv-9c37251f" to requested PersistentVolumeClaim due to incompatible volumeMode.
```

Reply:
duplicated by mistake. I removed it.